src/apps/administration/index.md

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import SwagLanding from "../../components/SwagLanding.vue";
+import SwagLandingCard from "../../components/SwagLandingCard.vue";
+import SwagLandingCardList from "../../components/SwagLandingCardList.vue";
+import * as HeroIcons from "@heroicons/vue/24/solid";
```

Comment:
We want to either make those icons globally available (once and for all), or import them separately. We'll take a look at this tomorrow.

Reply:
removed!